lib/diagnostic.js

```diff
@@ -9,14 +9,24 @@
 // Write a callback for reduce that selects
 // the minimum value in the array
 const min = function min() {
-
+  let smallest = arguments[0];
+  for (let i = 1; i < arguments.length; i++) {
+    if (smallest > arguments[i]) {
+      smallest = arguments[i];
+    }
+  }
+  return smallest;
```

Comment:
You misunderstood this question; we were asking you to create a function, `min`, that we could pass into `reduce` as a callback argument.
i.e.
```
[1,2,3].reduce(min);
// => 1
```